source/vir/Cargo.toml

```diff
@@ -10,6 +10,8 @@ edition = "2018"
 [dependencies]
 air = { path = "../air" }
 sise = "0.6.0"
+sha2 = "0.10.2"
+num-bigint = "0.4.3"
```

Comment:
The hashing technique introduces two dependencies but the alternative of using a counter to assign an id to a string could be non deterministic. 

Perhaps the middle ground could be to use a 64 bit (or perhaps even 128 is fine) hash and then increment the integer until a collision is resolved.

Reply:
num-bigint is already used in the interpreter so this dependency is fine, I have decided to proceed with the approach as a result.

Reply:
👍